translate/rus.json

```diff
@@ -114,7 +114,7 @@
 "duration":"длительность",
 "eco":"eco",
 "elo":"эло",
-"Enable Twitch channel":"Включить канал Twitch",
+"Enable Twitch channel":"Включить трансляцию Twitch",
```

Comment:
Same as previous. The last one meant 'turn the Twitch channel on'. I find it a little confusing for new users, so I changed it to 'turn the Twitch love broadcast on'.